python/scraper.py

```diff
@@ -128,7 +128,8 @@ def get_details(input_uniqueids, handle, settings):
             details['_info']['set_tmdbid'])
         details = combine_scraped_details_available_artwork(details,
             fanarttv_info,
-            settings.getSettingString('language'))
+            settings.getSettingString('language'),
+            settings)
```

Comment:
it's strange to transfer parament settings and settings.getSettingString('language'), may just one parament settings, or two parament settings.getSettingString('language') and settings.getSettingBool('prioritize_fanarttv_artwork')